modules/openapi-generator/src/main/resources/python/api_response.mustache

```diff
@@ -1,25 +1,21 @@
 """API response object."""
 
 from __future__ import annotations
-from typing import Any, Dict, Optional
-from pydantic import Field, StrictInt, StrictStr
+from typing import Any, Dict, Optional, Generic, TypeVar
+from pydantic import Field, StrictInt, StrictStr, BaseModel
 
-class ApiResponse:
```

Comment:
Making ApiResponse Pydantic and ApiResponse.data generic was one of the things I most wanted to do. This makes it more type safe.